lab5.md

```diff
@@ -63,7 +63,7 @@ Since we now have a new URL for ngrok let's save login details for the new URL:
 
 * Run `export PASSWORD="value-here"
 
-* Run `export NGROK_ADDRESS=https://fuh83fhfj.ngrok.io`
+* Export the tunnel urls as found in http://127.0.0.1:4040/inspect/http. For example: `export NGROK_ADDRESS=https://fuh83fhfj.ngrok.io`
```

Comment:
We already covered this in the note:

> Use the built-in UI of `ngrok` at http://127.0.0.1:4040 to find your HTTP URL. You will be given a URL that you can access over the Internet, it will connect directly to your OpenFaaS API Gateway.